src/autocomplete/dataset.js

```diff
@@ -93,7 +93,7 @@ _.mixin(Dataset.prototype, EventEmitter, {
     var renderArgs = [].slice.call(arguments, 2);
     this.$el.empty();
 
-    hasSuggestions = suggestions && suggestions.length;
+    hasSuggestions = suggestions && suggestions.length > 0;
```

Comment:
I think I got a plunkr together showing the bug for #178 here:  http://plnkr.co/edit/dQ5VzOX4EwiSJQWOCUoD?p=preview